free-1.md

```diff
@@ -10,41 +10,79 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Since Twitch Plays Pokemon began in 2014, there has been lots of interest
+in creating similar "crowdplay" games. However, it is an area primarily
+done by people willing to overcome the steep curve of learning how to interface
+with emulators and Twitch's API. A DSL could help people who would have previously
```

Comment:
This seems like a really interesting domain and a DSL seems like a reasonable solution for making the creation of Twitch Plays more accessible. I wonder if mediating this learning curve will rely more on language design or on creating a more intuitive interface, though I can't speak much on this since I've never used Twitch before.